test/kwokctl/kwokctl_workable_test.sh

```diff
@@ -132,6 +132,19 @@ function test_kwok_controller_port() {
     fi
 }
 
+function test_etcd_port() {
+    local result
+    nc -w 1 -z 127.0.0.1 2400
```

Comment:
```
$ curl http://127.0.0.1:2400/health
{"health":"true","reason":""}
```

I remember this port is for http, maybe we curl it directly...

Reply:
The problem is when the runtime is kind. Kind will start the etcd cluster with the certificate, causing http or https request to fail on a handshake... 
Not sure how to work around that issue without making it overly complicated